client/extensions/woocommerce/components/list/list-rows/index.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
This allows us to fully abstract out the table html and also style the th within this component.  If we do not have this component then we would need to style the tbody somewhere else.